ODLA/platforms/odla_popart/CMakeLists.txt

```diff
@@ -24,10 +24,11 @@ find_library(popart NAMES popart PATHS ${POPART_ROOT} PATH_SUFFIXES lib
              NO_DEFAULT_PATH)
 find_library(custom_ops NAMES custom_ops PATHS ${CMAKE_CURRENT_SOURCE_DIR}/custom_ops
              PATH_SUFFIXES lib NO_DEFAULT_PATH)
+find_package(popart CONFIG REQUIRED)
```

Comment:
Some build error:

https://github.com/alibaba/heterogeneity-aware-lowering-and-optimization/pull/142/checks?check_run_id=1722335369